ci.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Contributors to the Parsec project.
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
# Copyright 2021 Contributors to the Parsec project.
# SPDX-License-Identifier: Apache-2.0
```

Also, a comment with this license identifier should be present in every hand-written source file